src/figure/formation.c

```diff
@@ -761,13 +765,14 @@ void formations_save_state(buffer *buf, buffer *totals)
         buffer_write_u8(buf, f->herd_direction);
         buffer_skip(buf, 17);
```

Comment:
Small hint: since there are unused bytes in the formation buffer, you can likely use them (they will probably be set to 0 but you can confirm) instead of adding bytes at the end.

That way you don't need to bump the savegame version, as old saves will have 0 (which is harmless, no herd targeted) and new saves will have the target id, which will be ignored ("skipped") on old versions of Augustus.

Reply:
Makes sense to me. I’ll change it around when I get to fixing ranged units.